docs/pipeline-components-and-applications/enrichment-components/stream-enrich/index.md

```diff
@@ -1,12 +1,17 @@
 ---
-title: "Stream Enrich"
+title: "stream-enrich"
 date: "2020-11-09"
 sidebar_position: 800
 ---
 
+```mdx-code-block
+import {versions} from '@site/src/componentVersions';
+import CodeBlock from '@theme/CodeBlock';
+```
+
 :::note
 
-Stream Enrich will soon be deprecated in favor of [enrich-kinesis](/docs/pipeline-components-and-applications/enrichment-components/enrich/index.md). We recommend switching when you have the opportunity.
+`stream-enrich-kinesis` will soon be deprecated in favor of [enrich-kinesis](/docs/pipeline-components-and-applications/enrichment-components/enrich-kinesis/index.md). We recommend switching when you have the opportunity.
```

Comment:
Should we add any note about the others? Or at least Kafka.

Reply:
I don't think so. We announce the deprecation to encourage users to migrate to `enrich-kinesis` already. Other assets (well, Kafka mostly) don't exist yet so people should continue to use `Stream Enrich Kafka` until the other asset is ready.